Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -20,12 +20,15 @@ class CustomerInfoResponseHandler {
     func handle(customerInfoResponse response: HTTPResponse<Response>.Result,
                 completion: CustomerAPI.CustomerInfoResponseHandler) {
         let result: Result<CustomerInfo, BackendError> = response
-            .flatMap {
-                if $0.body.errorResponse.attributeErrors.isEmpty {
-                    return .success($0.body.customerInfo)
-                } else {
-                    return .failure(.errorResponse($0.body.errorResponse, $0.statusCode))
+            .map {
+                // If the response was successful we always want to return the `CustomerInfo`.
+                if !$0.body.errorResponse.attributeErrors.isEmpty {
+                    // If there are any, log attribute errors.
+                    // Creating the error implicitly logs it.
+                    _ = $0.body.errorResponse.asBackendError(with: $0.statusCode)
```

Comment:
I agree. I didn't feel _too_ bad about this instance because it's covered by a test, so removing it would make it fail.
I don't love the implicit logging, but I think explicit logging of errors everywhere would be even worse :/ so I'm not sure how to improve this at the moment.

Reply:
Yeah, that's a great point, as long as it's covered by a test, this is much more palatable 😄